src/foam/nanos/boot/Boot.java

```diff
@@ -42,6 +44,8 @@ public Boot() {
   public Boot(String datadir) {
     Logger logger = new ProxyLogger(new StdoutLogger());
     root_.put("logger", logger);
+    root_.put("pmInfoDAO", new foam.nanos.pm.PMInfoMDAO());
```

Comment:
This shouldn't be necessary.  I would rather find out why this isn't working and fix it than hard-code PM's into Boot. Maybe we need a localPMInfoDAO which isn't authenticated or served, which is used by the PMLogger. Would be faster as well.

Reply:
The real fix, which was hidden my other troubleshooting was explicitly creating the PM rather than rely on the PM factory which just generates 1% of non null PMs.